### PR TITLE
Reimplement automatic achievements + add 2 more

### PR DESCRIPTION
**Pull request by JunoLee128**
- fixed some errors when trying to add achievements/icons
- officership/event/etc. achievements should get automatically assigned again
- added Bee Enthusiast, and Week of TBP achievements
    - still need to add to production using Django admin. I don't know how to SSH, to access the media files?

### CODE REVIEW

File: achievements/apps.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+class AchievementConfig(AppConfig):
+    name = 'achievements'
+    verbose_name = 'Achievements'
+    def ready(self):
+        import achievements.course_file_achievements
+        import achievements.event_achievements
+        import achievements.meta_achievements
+        import achievements.officership_achievements
```
Comment (ochan1) [inline achievements/apps.py:10]: Add a whitespace at the end

Comment (ochan1) [inline achievements/apps.py:10]: Add a whitespace at the end

**Comment by ochan1** [comment]
In general, please remember to use Unix style whitespacing

If using VS Code, more info to setup here: https://medium.com/bootdotdev/how-to-get-consistent-line-breaks-in-vs-code-lf-vs-crlf-e1583bf0f0b6, please use the LF setting

For now, this is fine
If you want to find a way to use the above setting to fix the newline whitespace issue, great!

Otherwise, you can wait until I look at the PR and I'll probably help by making another commit to fix the whitespace issue

**Comment by snowshoes7** [comment]
other than the whitespace stuff i think it should be good, tested locally

File: notifications/models.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ class Notification(models.Model):
                    'about the notification (e.g. how an achievement is '
                    'unlocked or how long a project report is overdue).'))
     image_url = models.CharField(
-        blank=True,
+        blank=False,
```
Comment (ochan1) [inline notifications/models.py:40]: Please clarify why this 'image_url' is set to being a required field (in other words, must be filled in as 'blank' being False (meaning CAN'T be blank))

**Review by ochan1 — CHANGES_REQUESTED** [review]
Juno, please approve the changes and bug fix made to your branch here: https://github.com/JunoLee128/tbpweb/pull/1

Then everything should be good

File: notifications/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ochan1) [reply notifications/models.py:40]: From Juno via Slack: for the blank=False, I was getting an error when it the field wasn't required, because it might have been set to None. blank=True, null=True might work too

**Comment by ochan1** [comment]
First make the Achievement, and then you can upload the icon
Achievement Icon uploads can be done in the TBP Admin under https://tbp.berkeley.edu/admin/achievements/achievementicon/

You don't need to SSH into the server

**Comment by ochan1** [comment]
After some investigation, for the Notifications to work, you need to have the Achievement Icon in Static folder (in addition to Media folder)

This isn't the most ideal solution ... but long term should be from the Media folder
But Achievements don't get added left and right, and non-Achievement icons also exists also, so some logic as a feature may be needed

TLDR for now: Need to put icon in both Static (can be through pushed code) and uploaded to Achievement Icon, still don't need SSH
